testsuite/features/reposync/srv_wait_for_reposync.feature

```diff
@@ -1,4 +1,4 @@
-# Copyright (c) 2019-2022 SUSE LLC
+# Copyright (c) 2019-2023 SUSE LLC
```

Comment:
since you didn't touch this file maybe restore this as well, but completely optional